ros/finalarm_simplified_control/CMakeLists.txt

```diff
@@ -9,11 +9,3 @@ find_package(roslaunch)
 
 install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
 install(DIRECTORY config DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
-install(DIRECTORY script DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
I removed these scripts from the finalarm_simplified_control and delegate them to the finalarm_control. I dont think we should duplicate the scripts across the modules.